### DISCUSSION TITLE
[Script request] Defguard

### DISCUSSION THREAD

**Discussion by Aloe-recite** [community-scripts/ProxmoxVE#394] [Request script]
### Application Name

Defguard

### Website

https://docs.defguard.net/

### Description

Defguard is a comprehensive Remote Access Management solution incorporating in one solution:
- Remote Access secured by [WireGuard® VPN with 2FA/Multi-Factor Authentication](https://docs.defguard.net/admin-and-features/features-and-configuration/wireguard),
- Identity Management with [SSO based on OpenID Identity Provider](https://docs.defguard.net/admin-and-features/features-and-configuration/openid-connect),
- Account Lifecycle management with [secure remote account onboarding](https://docs.defguard.net/help/enrollment).

The project looks good since all enterprise features up to the following limits are free and [no license is required](https://docs.defguard.net/enterprise/license), which seems good for a home or even small office setup.
- 5 active users
- 10 devices
- 1 location

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by yodaphone** [comment]
+1. Hope someone can create a script for this.

**Comment by MickLesk** [comment]
https://github.com/community-scripts/ProxmoxVED/issues/2111
